xml/admin_ceph_datamgm.xml

```diff
@@ -1472,59 +1472,55 @@ osdmap e13 pg 1.6c (1.6c) -> up [1,0] acting [1,0]
     <sect2 xml:id="op-pgs-autoscaler">
      <title>PG Auto-scaler</title>
      <para>
-       As of the Nautilus release, &ceph; includes a new manager module called
-       <literal>pg_autoscaler</literal> that allows the cluster to consider the
-       amount of data actually stored (or expected to be stored) in each
-       pool and choose appropriate <option>pg_num</option> values automatically.
+       Placement groups (PGs) are an internal implementation detail of how
+       &ceph; distributes data. You can allow the cluster to either make
+       recommendations or automatically tune PGs based on how the cluster
+       is used by enabling pg-autoscaling.
```

Comment:
```suggestion
       &ceph; distributes data.  By enabling pg-autoscaling, you can allow
       the cluster to either make or automatically tune PGs based on how
       the cluster
       is used.
```

Reply:
Just a suggestion -- I think putting what it does up front before the long explanatory sentence is clearer. 

I am happy to be voted down on this if others disagree.